src/components/screen/SearchUser.js

```diff
@@ -25,35 +26,80 @@ import { ProfileModalConsumer } from '../../providers/ProfileModalProvider';
 import EmptyListItem from '../shared/EmptyListItem';
 import UserListItem from '../shared/UserListItem';
 
-type Styles = {
-  container: ViewStyle,
-  viewSearch: ViewStyle,
-  imgSearch: ImageStyle,
-};
+import styled from 'styled-components/native';
 
-const styles: Styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    backgroundColor: 'transparent',
-    flexDirection: 'column',
-    alignItems: 'center',
-    justifyContent: 'center',
+const FriendsSampleData = [
```

Comment:
It's not `FriendsSampleData`, it should be `UserSampleData`.
Also, I think it's not a good idea to use `upper camel case` with a variable (not a class or type).
If you really want it to be capital, you better name it like `USER_SAMPLE_DATA`. I recommend this to be `userSampleData`.

Reply:
okay. I renamed it